lib/manifestTools/assets/web-manifest.json

```diff
@@ -46,8 +46,8 @@
       "properties": {
         "density": {
           "description": "The density member of an icon is the device pixel density for which this icon was designed.",
-          "type": "number",
-          "default": 1.0
+          "type": [ "number", "string" ],
+          "default": [ 1.0, "one" ]

```

Comment:
Difficult for me to tell if this late commit went through a few minutes ago. Git says that it did not, but the last time it told me the same thing, it worked.